src/btf.h

```diff
@@ -47,6 +47,10 @@ static const std::vector<std::string_view> RT_BTF_PREFIXES = { "__probestub_",
 
 class BPFtrace;
 
+// Prefix used to generate names for anonymous struct/unions. The
+// format is: BTF_ANON_STRUCT_PREFIX + <btfid> + "_" + <btfobjname>
+const std::string BTF_ANON_STRUCT_PREFIX = "__bpftrace_btf_anon_";
```

Comment:
Should be more efficient:
```suggestion
constexpr std::string_view BTF_ANON_STRUCT_PREFIX = "__bpftrace_btf_anon_";
```

Reply:
Right, and none of those are involved in a '+' operation. So, I could change it to ```constexpr std::string_view```, I'd just have to convert the addition in ```create_anon_btf_name``` to convert it to std::string before doing the addition. Which is done with e.g. ```TRACEPOINT_STRUCT_PREFIX```

So, seems like a wash as to what would be better, perhaps? I can change it though, if definitions like this are all std::string_view, to be consistent.

Reply:
Ok, I made the change + conversion to be consistent with other prefix defines.

Reply:
Ah, right. I think it makes sense to be consistent here.